apps/api/src/domain/models/vocabulary-list/entities/vocabulary-list.entity.ts

```diff
@@ -112,6 +128,44 @@ export class VocabularyList extends Resource {
         });
     }
 
+    // TODO should type be an enum?
```

Comment:
[Here](https://www.pivotaltracker.com/story/show/186469235) is a related story. Do this at the same time.